src/lib/sidenav/drawer.ts

```diff
@@ -325,6 +328,27 @@ export class MdDrawerContainer implements AfterContentInit {
   /** Event emitted when the drawer backdrop is clicked. */
   @Output() backdropClick = new EventEmitter<void>();
 
+  /**
+   * Width of the container at which it breaks
+   * e.g., breakpointWidth="500"
+   */
+  @Input() breakpointWidth: number;
+
+  /** Whether the drawer changes modes when collapsing. */
+  @Input()
+  get breakpointChangeMode(): boolean { return this._breakpointChangeMode; }
+  set breakpointChangeMode(value: boolean) {
+    this._breakpointChangeMode = coerceBooleanProperty(value);
+  }
+  private _breakpointChangeMode: boolean = true;
+
+  /**
+   * Responsively toggle the drawer using the breakpoint.
+   * Note that this only works when the window is rezied.
+   * If you resize it some other way, call _updateDrawer().
+   */
+  _updateDrawer: Subscription | null = null;
```

Comment:
Maybe rename this to `_updateDrawerSubscription` and then pull out a method named `_updateDrawer` that can still manually be used? Or just rename it and update the comment.